staging/src/k8s.io/kubectl/pkg/cmd/expose/expose.go

```diff
@@ -147,20 +172,17 @@ func NewCmdExposeService(f cmdutil.Factory, streams genericclioptions.IOStreams)
 	o.RecordFlags.AddFlags(cmd)
 	o.PrintFlags.AddFlags(cmd)
 
-	cmd.Flags().String("generator", "service/v2", i18n.T("The name of the API generator to use. There are 2 generators: 'service/v1' and 'service/v2'. The only difference between them is that service port in v1 is named 'default', while it is left unnamed in v2. Default is 'service/v2'."))
```

Comment:
Seem like version 1 generator has been deprecated https://github.com/kubernetes/kubectl/blob/b1d9ef54819c3666364d2a0ea49cf3f7faf7454f/pkg/generate/versioned/generator.go#L40

Reply:
Yeah, it was removed earlier this year in https://github.com/kubernetes/kubernetes/pull/99733